src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -265,7 +269,7 @@ private function extractContentFromAnnotation(Annotation $annotation)
             return '';
         }
 
-        $content = $matches[1];
+        $content = Preg::replace('/\*+\/$/', '', $matches[1]);
```

Comment:
You need to match exactly 2 stars followed by a space. One star is not phpdoc, it's a comment.

Reply:
That applies to the beginning of the comment, this RegEx is to match at least one `*`, followed by `$` at the end of the comment.

Reply:
Ah, yes, I misread those slashes at the start.

Reply:
Anything else regarding reviewing this then?